models/endpoints.model.js

```diff
@@ -0,0 +1,5 @@
+const endpointsJSON = require("../endpoints.json");
+
+exports.fetchEndpoints = () => {
+  return endpointsJSON;
+};
```

Comment:
Looks great Helen, I like how you've split the functionality out into it's own model and controller files to keep it organised 📝